nerfstudio/data/dataparsers/nerfosr_dataparser.py

```diff
@@ -83,13 +83,12 @@ def get_camera_params(scene_dir: str, split: str) -> Tuple[torch.Tensor, torch.T
 
     Args"
       scene_dir : The directory containing the scene data.
-      split : The split for which to load the camera parameters. Either "train", "val", or "test".
+      split : The split for which to load the camera parameters. Either "train", "validation", or "test".
```

Comment:
I missed this earlier, but in the definition change it to,
`def get_camera_params(scene_dir: str, split: Literal["train", "validation", "test"]) -> ...`
And then `Either "train", "validation", or "test".` can be removed from the docstring.